src/utils/fs.spec.js

```diff
@@ -1,39 +1,189 @@
+const fs = require('fs')
+const path = require('path')
+
+jest.mock('fs')
+jest.mock('path')
+jest.mock('util', () => ({
+  promisify: jest.fn().mockImplementation(fn => fn),
```

Comment:
I'll never learn which Jest methods are aliases and which ones are different variations, but I think this could be written as 
```suggestion
  promisify: jest.fn(fn => fn),
```